src/libs/antares/study/parts/short-term-storage/series.cpp

```diff
@@ -127,15 +133,19 @@ bool Series::saveToFolder(const std::string& folder) const
 
     bool ret = true;
 
-    auto checkWrite = [&ret, &folder](const std::string& name, const std::vector<double>& content) {
-        ret = writeVectorToFile(folder + SEP + name, content) && ret;
+    auto checkWrite = [&ret, &folder]
+        (const std::string& filename, const std::vector<double>& timeSeries, double defValue)
+    {
+        // if vector is only default values we don't save it
+        if (!hasOnlyDefaultValues(timeSeries, defValue))
+            ret = writeVectorToFile(folder + SEP + filename, timeSeries) && ret;
```

Comment:
Function **writeVectorToFile** should be renamed into **writeTimeSeriesToFile**.